### DISCUSSION TITLE
Trigger one page rebuild from custom express server

### DISCUSSION THREAD

**Discussion by arso1er** [vercel/next.js#11698] [Help] [answered]
Hi. 

I've read the documentation about getStaticProps and getStaticPaths for statically pre-rendering pages that use dynamic routes. I'd like to know if there is a way to rebuild only one page when it's updated. For example let's say I'm rendering product pages, and a product can be updated via a call to my custom express api. Could I trigger a rebuild of the updated product page from my api (something like app.buildOne('/products/my-product-id'))?

Thanks.

**Comment by timneutkens** [comment, marked as answer]
This is covered by this RFC: https://github.com/zeit/next.js/discussions/11552

**Comment by arso1er** [comment]
Great. Thanks for your answer.

**Comment by acalvino4** [comment]
@timneutkens
That RFC does mention the feature which @arso1er asked about

> After this RFC for incremental static generation is implemented we can extend it to do force revalidation, which would be useful for CMS providers that want to get the content updated as quickly as possible.

but as far as I can tell, it was not covered in the implementation of this RFC, rather this is just a note of what can be done next: "After this RFC..."
Is this correct, and if so, is there any word on this feature in in the product roadmap?

Thanks!

**Reply by cirosantilli** [reply]
I have now created a dedicated discussion to this/something closely related at: https://github.com/vercel/next.js/discussions/25677 Edit: OK, thinking about it, it is not the same as this, but something complementary.
